README.md

```diff
@@ -60,8 +60,19 @@ __Please note that you must use [express-session](https://github.com/expressjs/s
 * `cookie.name` String - Required if cookie is an object and `angular` is not true. The CSRF cookie name to set.
 * `cookie.options` Object - Optional. A valid Express cookie options object.
 * `angular` Boolean - Optional. Shorthand setting to set `lusca` up to use the default settings for CSRF validation according to the [AngularJS docs]. Can be used with `cookie.options`.
-* `blocklist` Array or String - Optional. Allows defining a set of routes that will not have csrf protection.  All others will.
-* `allowlist` Array or String - Optional. Allows defining a set of routes that will have csrf protection.  All others will not.
+* `blocklist` Array or String - Optional. Allows defining a set of routes that will not have csrf protection.  All others will.  
+Example configuration:
+  ```
+  blocklist: [{path: '/details', type: 'exact'}, {path: '/summary', type: 'includes'}]
+  //If match type is 'exact', route will get blocklisted only if it matches req.path exactly
+  //If match type is 'includes', Lusca will check if req.path starts with the specified path
```

Comment:
For a sec, i. confused with the includes method which searches the occurrence irrespective of position.
 includes isn't same as startWith. 

{path: '/summary', type: 'includes'} => /details/summary 
{path: '/summary', type: 'startsWith'} => /summary/details. 


if we just want to check req.path starts with the specified path..then we shouldn't keep the type as `includes`.

Reply:
Looks like it should be startsWith

Reply:
Makes sense. Updated.